lib/manager/pip-compile/artifacts.ts

```diff
@@ -57,7 +58,7 @@ export async function updateArtifacts({
     if (config.isLockFileMaintenance) {
       await deleteLocalFile(outputFileName);
     }
-    const cmd = 'pip-compile';
+    const cmd = `pip-compile ${parse(inputFileName).base}`;
```

Comment:
Should we quote this? Will this work with subdirectories?

Reply:
Yes, it returns the file name even when in a subdirectory. We want the filename without path because we already set `cwdFile`